src/docs/product/error-monitoring/issue-details/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Issue Details
+sidebar_order: 10
+description: "Learn more about what information displays on the Issue Details page to help you efficiently triage an issue."
+---
+
+The Issue Details page helps you to gain further insight into the source of the issue. The information that is provided on the primary Issues page also displays on the Issue Details page. This page displays information about a specific error event that's part of an issue, however, you can navigate between the events of an issue using the "Older" and "Newer" buttons.
+
```

Comment:
What does `The information that is provided on the primary Issues page also displays on the Issue Details page` mean? I think we should cut that.